src/pages/groups/List.js

```diff
@@ -196,29 +213,32 @@ const GroupList = () => {
         value: filter
       })
     ]),
-    h(PageBox, { role: 'main', style: { flexGrow: 1 } }, [
+    h(PageBox, { role: 'main', style: { flexGrow: 1, backgroundColor: colors.light(), margin: 0, padding: '3rem 3rem 1.5rem' } }, [
       div({ style: Style.cardList.toolbarContainer }, [
         div({ style: { ...Style.elements.sectionHeader, textTransform: 'uppercase' } }, [
           'Group Management'
         ])
       ]),
-      div({ style: Style.cardList.cardContainer }, [
+      div({ style: { marginTop: '1rem' } }, [
         h(NewGroupCard, {
           onClick: () => setCreatingNewGroup(true)
         }),
         Utils.cond(
           [groups && _.isEmpty(groups), () => noGroupsMessage],
           [!_.isEmpty(groups) && _.isEmpty(filteredGroups), () => {
-            return div({ style: { fontStyle: 'italic' } }, ['No matching groups'])
+            return div({ style: { fontStyle: 'italic', marginTop: '1rem' } }, ['No matching groups'])
           }],
           () => {
-            return div({ style: { flexGrow: 1 } }, [
-              _.map(group => {
-                return h(GroupCard, {
-                  group, key: `${group.groupName}`,
-                  onDelete: () => setDeletingGroup(group)
-                })
-              }, filteredGroups)
+            return h(Fragment, [
+              h(GroupCardHeaders, { sort, onSort: setSort }),
+              div({ style: { flexGrow: 1, marginTop: '1rem' } }, [
```

Comment:
@andrewmarcus any thoughts here?

Reply:
Re: the column with the roles being pushed right against the trash can… I am inclined to punt on that. This is an aside to the ticket, and is the existing behavior on the group page. I was mainly doing the header and styling on this page to ensure it stays in alignment, future changes/extra changes should probably have their own ticket. Any objections?

Reply:
Actually, the trashcan is in the same column as the role, so the A11Y question is actually handled now, so I am going to say that section of that is resolved.

Reply:
Hey there, thanks for asking. You definitely need a column heading on every column, although you can use the `sr-only` class to make it only available for screen readers and not visible if that makes sense. I haven't looked through the code, but in the second pair of screenshots above, you'll probably need a header over the column that has the values "Admin" and "Member".

I'm not sure about the extra menu toggle and delete button at the end of the row. Probably fine to leave them where they are rather than breaking them into their own column. I guess the only reason to give them a separate column would be that a screen reader users navigating the table will hear something like "Admin Delete, cell, row 3 column 3"

Reply:
That screen reader text there is compelling to me. Will break them out!